Makefile

```diff
@@ -44,20 +42,8 @@ PATH := $(GO_BIND_BIN_DIR):$(PATH)
 
 # * Main commands
 
-# `all` and `build` command builds everything (generate, build.ios, build.android)
-all build: generate build.ios build.android
-
-# Build iOS framework
-build.ios: generate framework.ios
```

Comment:
The README still says to use `make build.ios` and `make build.andriod`. Need to update the README?
https://github.com/D4ryl00/gnonative/blob/chore/move-gnoboard/README.md

Reply:
fixed, thanks.